### ISSUE TITLE
Panic when the user do not have the permission to delete pods

### ISSUE THREAD

**Issue by dozer47528** [aylei/kubectl-debug#38]
Start deleting agent pod debug-pod-585fcf9d59-njmwk
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x10c9a57]

goroutine 1 [running]:
fmt.Fprintf(0x0, 0x0, 0x20b845f, 0x4e, 0xc0007157f0, 0x2, 0x2, 0x39, 0x0, 0x0)
	/usr/local/Cellar/go/1.12.4/libexec/src/fmt/print.go:200 +0x77
github.com/aylei/kubectl-debug/pkg/plugin.(*DebugOptions).Run.func2.1()
	/Users/alei/go/src/github.com/aylei/kubectl-debug/pkg/plugin/cmd.go:472 +0x2c9
k8s.io/kubernetes/pkg/util/interrupt.(*Handler).Close.func1()
	/Users/alei/go/pkg/mod/k8s.io/kubernetes@v1.13.1/pkg/util/interrupt/interrupt.go:65 +0x46
sync.(*Once).Do(0xc0003ac080, 0xc000715888)
	/usr/local/Cellar/go/1.12.4/libexec/src/sync/once.go:44 +0xb3
k8s.io/kubernetes/pkg/util/interrupt.(*Handler).Close(0xc0003ac060)
	/Users/alei/go/pkg/mod/k8s.io/kubernetes@v1.13.1/pkg/util/interrupt/interrupt.go:63 +0x54
k8s.io/kubernetes/pkg/util/interrupt.(*Handler).Run(0xc0003ac060, 0xc000758c40, 0x0, 0x0)
	/Users/alei/go/pkg/mod/k8s.io/kubernetes@v1.13.1/pkg/util/interrupt/interrupt.go:103 +0x113
github.com/aylei/kubectl-debug/pkg/plugin.(*DebugOptions).Run.func2(0xc000000008, 0x20fac30)
	/Users/alei/go/src/github.com/aylei/kubectl-debug/pkg/plugin/cmd.go:475 +0xf3
k8s.io/kubernetes/pkg/util/interrupt.(*Handler).Run(0xc00076e300, 0xc00076e2d0, 0x0, 0x0)
	/Users/alei/go/pkg/mod/k8s.io/kubernetes@v1.13.1/pkg/util/interrupt/interrupt.go:103 +0xff
github.com/aylei/kubectl-debug/pkg/util.TTY.Safe(0x2244600, 0xc00000e010, 0x2244620, 0xc0000bc000, 0x1, 0x0, 0xc00015a2d0, 0xc00076e2d0, 0x0, 0x0)
	/Users/alei/go/src/github.com/aylei/kubectl-debug/pkg/util/term.go:110 +0x189
github.com/aylei/kubectl-debug/pkg/plugin.(*DebugOptions).Run(0xc000182c40, 0x0, 0x20fac28)
	/Users/alei/go/src/github.com/aylei/kubectl-debug/pkg/plugin/cmd.go:478 +0x8ea
github.com/aylei/kubectl-debug/pkg/plugin.NewDebugCmd.func1(0xc00037a280, 0xc00015a190, 0x2, 0x5)
	/Users/alei/go/src/github.com/aylei/kubectl-debug/pkg/plugin/cmd.go:150 +0xe7
github.com/spf13/cobra.(*Command).execute(0xc00037a280, 0xc0000be0d0, 0x5, 0x5, 0xc00037a280, 0xc0000be0d0)
	/Users/alei/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0xc00037a280, 0xc00000e010, 0x2244620, 0xc0000bc000)
	/Users/alei/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/alei/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
	/Users/alei/go/src/github.com/aylei/kubectl-debug/cmd/plugin/main.go:17 +0x110

**Comment by aylei** [comment]
@dozer47528 
Could you please provide the `kubectl-debug` version you are using? 
You can get the version by `kubectl debug --version`

**Comment by dozer47528** [comment]
```
$ kubectl debug --version
debug version v0.0.0-master+$Format:%h$
```

**Comment by aylei** [comment]
Weird... According to the stack trace the `ErrOut`(default to `os.Stderr`) is nil, what's your environment? (OS, shell... etc)

**Comment by dozer47528** [comment]
MacOS 10.14.5 (18F132)
oh-my-zsh

**Comment by calmkart** [comment]
fixed in #70 
@aylei 
PTAL, thanks

**Comment by aylei** [comment]
closed via #70
